addon/utils/matchers.js

```diff
@@ -49,12 +49,23 @@ export function matchingComponent(rawComponentName, path) {
   return result;
 }
 
+const templateStartRegExp = /^[\s\S]*([/\\]pods[/\\])/i;
```

Comment:
looks like pod prefix name may be different, and I think we already have some logic to resolve it, let's use it 
![image](https://user-images.githubusercontent.com/1360552/98800372-585de000-2421-11eb-9e27-914d6662e397.png)